### PR TITLE
Adds optionals section on README for step 4-projects

### PR DESCRIPTION
**Pull request by amandakarina**
Hey Folks

This PR highlights the toggles for optional services on step 4-projects

Can you take a look?

Thanks!

### CODE REVIEW

**Review by bharathkkb — APPROVED** [review]
LGTM 
/cc @rjerrems

File: 4-projects/README.md [hunk 1]
```diff
@@ -94,3 +94,14 @@ When using Cloud Build or Jenkins as your CI/CD tool each environment correspond
 1. Run `./tf-wrapper.sh apply development`
 
 If you received any errors or made any changes to the Terraform config or `terraform.tfvars` you must re-run `./tf-wrapper.sh plan <env>` before run `./tf-wrapper.sh apply <env>`
+
+### Optionals
+
+This module provides some optional projects configurations. You can check below what are they and what will be created if you enable each one:
+
```
Comment (rjerrems) [inline 4-projects/README.md:101]: We should remove these and auto generate as per feedback here:

https://github.com/terraform-google-modules/terraform-example-foundation/pull/239

**Review by rjerrems — CHANGES_REQUESTED** [review]
Overall looks good, if you remove the manually added docs for optionals should be good to merge.

File: 4-projects/README.md [hunk 2]
```diff
@@ -94,3 +94,7 @@ When using Cloud Build or Jenkins as your CI/CD tool each environment correspond
 1. Run `./tf-wrapper.sh apply development`
 
 If you received any errors or made any changes to the Terraform config or `terraform.tfvars` you must re-run `./tf-wrapper.sh plan <env>` before run `./tf-wrapper.sh apply <env>`
+
+### Optional Variables
+
+Some variables used to deploy the step have default values, check if you want to change those values before doing the deploy. For more information check in the section **Inputs** the variables not required for the environments [development](./business_unit_1/development/README.md#Inputs), [non-production](./business_unit_1/non-production/README.md#Inputs) and [production](./business_unit_1/production/README.md#Inputs).
```
Comment (rjerrems) [inline 4-projects/README.md:100]: nit: If @atos-cit agrees with my feedback here https://github.com/terraform-google-modules/terraform-example-foundation/pull/239, I think we can remove this.

**Review by rjerrems — APPROVED** [review]
One small nit, but otherwise LGTM

File: 4-projects/README.md (hunk 2 above)
Comment (atos-cit) [reply 4-projects/README.md]: Removed the section as agreed in #239 .

**Pull request merged**